lib/assessment.js

```diff
@@ -8,28 +8,66 @@
 // Question 1
 // Write a callback for reduce that selects
 // the minimum value in the array
+<<<<<<< HEAD
+var min = function min(array, callback, initialValue) {
+  var previousValue;
+  if (initialValue !== undefined) {
+    previousValue = initialValue;
+  } else {
+    array = array.slice();
+    previousValue = array.shift();
+  }
+
+  array.forEach(function(currentValue, index, array) {
+    previousValue = callback(previousValue, currentValue, index, array);
+  });
+
+  return previousValue;
+};
+
+var prod = function prod(previousValue, currentValue, index, array) {
+  return previousValue * currentValue;
+=======
 var min = function min(previousValue, currentValue) {
   return previousValue < currentValue ? previousValue : currentValue;
+>>>>>>> origin/solution
 };
 
 // Question 2
 // Write a callback for reduce that produces
 // the sum of the values in the array
+<<<<<<< HEAD
+var sum = function sum(numbers) {
+    var total = 0;
+for ( var i = 0; i < numbers.length; i++ ){
+    total += numbers[i];
+}
+    return total;
+}
```

Comment:
`sum` will compute the sum of the array elements effectively when passed the array. It works as a standalone function that takes an array as its only argument.

Q2 asked for a function that gets passed in to `Array.reduce` to operate on at least two arguments, `previousValue` and `currentValue`. It should not loop over anything in its body, as `Array.reduce` is already iterating over the array.

Half credit for writing a good standalone function to do the desired task.